### PR TITLE
Update fancybox package.json and add v3.0.39 w/ git auto-update

### PR DESCRIPTION
**Pull request by maruilian11**
@amomo could you help me check this PR for #6191? thank you!

### CODE REVIEW

**Comment by Amomo** [comment]
Don't we minify `jquery.fancybox-thumbs.css` and `jquery.fancybox-thumbs.js`?

**Comment by maruilian11** [comment]
@Amomo updated~

**Comment by Amomo** [comment]
@PeterDaveHello I think this PR is ok, could you check again? thanks~

**Comment by PeterDaveHello** [comment]
Didn't you try to confirm the version as git tag in the repo?

The text says Beta 1 here: http://fancyapps.com/fancybox/beta/, it's better to confirm with author first.

BTW, the license on the website also looks different with the one on CDNJS:
http://fancyapps.com/fancybox/#license

**Comment by maruilian11** [comment]
wait for https://github.com/fancyapps/fancyBox/issues/1133

**Comment by maruilian11** [comment]
@cdnjs/intern2 Please help me review this PR, thank you!

**Comment by maruilian11** [comment]
@fancyapps sorry about not noticing that :sweat_smile: 
 I just updated it.

**Comment by maruilian11** [comment]
@cdnjs/intern2 Please help me review this PR, thanks.

**Comment by PeterDaveHello** [comment]
The commercial license is still in readme, @maruilian11 have you confirmed that it should be removed?
@fancyapps what do you think? I think GPL-3.0 licensed project can be used in commercial scenario?

**Comment by fancyapps** [comment]
I think you can leave `"license": "GPL-3.0"` field as this seems a common practice for projects with dual licensing (and including details about licensing in readme or license file) since npm has prohibited using "SEE LICENSE IN <filename>".

**Comment by PeterDaveHello** [comment]
So maybe we should just leave the url to the license page, add GPL-3.0 there.

**Comment by maruilian11** [comment]
@PeterDaveHello How about we leave the url, and also add this page: http://fancyapps.com/fancybox/3/#license

**Comment by PeterDaveHello** [comment]
Let's add both GPL v3 and that url, thanks.

**Comment by maruilian11** [comment]
@cdnjs/intern2 Please help me review this PR, thanks.

File: ajax/libs/fancybox/package.json [hunk 1]
```diff
@@ -12,13 +12,25 @@
     "zoom",
     "zooming"
   ],
-  "namespace": "$.fancybox",
```
Comment (PeterDaveHello) [inline ajax/libs/fancybox/package.json]: please leave this alone

File: ajax/libs/fancybox/package.json [hunk 2]
```diff
@@ -12,13 +12,25 @@
...
   "repository": {
     "type": "git",
-    "url": "https://github.com/fancyapps/fancyBox"
+    "url": "https://github.com/fancyapps/fancybox"
   },
-  "license": {
-    "type": "fancyBox",
-    "url": "http://fancyapps.com/fancybox/#license"
+  "licenses": [
+    "http://fancyapps.com/fancybox/#license",
```
Comment (PeterDaveHello) [inline ajax/libs/fancybox/package.json]: custom license need to be an object, not string

**Comment by maruilian11** [comment]
@cdnjs/intern2 Please help me review this PR, thanks.

**Comment by PeterDaveHello** [comment]
@maruilian11 mention the fields you updated in the commit message next time please.

**Comment by maruilian11** [comment]
got it.

**Pull request merged**